.github/workflows/unit-test.yml

```
@@ -38,5 +38,28 @@ jobs:
         with:
           go-version: "1.21"
 
+      - name: Setup jq
+        uses: dcarbone/install-jq-action@v2.0.2
+        with:
+          version: |
+            type: string
+            required: false
+            description: "Version of jq to install"
+            default: "1.7"
```

The pipe (`|`) in yaml syntax means to treat the below block as a multi-line string, which does not match what I see documented: https://github.com/dcarbone/install-jq-action#version

I'm not certain, but this has a decent probability of changing the meaning entirely compared to what it seems to be doing.

```suggestion
          version:
            type: string
            required: false
            description: "Version of jq to install"
            default: "1.7"
```